bubble_sort.py

```diff
@@ -1,8 +1,3 @@
 def sorter(arr):
-    for i in range(len(arr)):
```

Comment:
Suggested change: Use sort() instead of custom logic

Suggested change:
```diff
- def sorter2(arr):
+ def sorter2(arr):
+     arr.sort()
+     return arr